pyndl/preprocess.py

```diff
@@ -211,7 +233,26 @@ def create_event_file(corpus_file,
         be (three) consecutive words, a sentence, or a line in the corpus file.
 
     """
-    if '_' in symbols or '#' in symbols or '\t' in symbols:
+    if isinstance(symbols, str):
+        not_in_symbols = re.compile(f"[^{symbols:s}]")
+        def filter_symbols(line, replace):
+            return not_in_symbols.sub(replace, line)
+    elif isinstance(symbols, re.Pattern):
+        not_in_symbols = symbols
```

Comment:
I don't think that this part here is correct. From my understanding, the logic of the code is the following:

- symbols describe which characters should be included
- a regex is crafted that defines all characters that are not in symbols (cf. line 237)
- this regex is used to remove all the characters, by running `filter_symbols(line, ' ')` in line 321

Therefore, I believe the regex of all valid symbols would need to be negated here

This also seems not to be covered by the tests we have. However, running 
``` python
create_event_file(RESOURCE_FILE, event_file,
                      context_structure="document",
                      event_structure="word_to_word", event_options=(2, 1),
                      cue_structure="word_to_word")
```
raises a 
```
ValueError: "_", "#", and "\t" are special symbols and cannot be in symbols string
```

Reply:
As Github shows a odd preview of the code here: By "this part" I mean 240ff

Reply:
If `symbols` is a string we compile a re.Pattern that finds all symbols that are NOT in the symbols. But as there is no easy way to negate a regular expression (as far as my 5 minutes googling came), I opted to treat the symbols parameter as a `not_in_symbols` re.Pattern. This is counter intuative and I am happy about another solution.

I will add a test from the above code.

Reply:
But in that case symbols would need to be a regex of all symbols that should be excluded (`"[#_\t]" instead of "[^#_\t]"), but this is indeed very counterintuitive..

Reply:
I added the test but have to do other stuff right now. You are right and the code fails.

Reply:
This discussion is resolved by the following commit [8d0ddf5](https://github.com/quantling/pyndl/pull/224/commits/8d0ddf5761796d6b50ea8e85c115d3c5e6ea91d1), right?